src/types/options/statisticsRequestOptions.ts

```diff
@@ -1,25 +1,25 @@
-import { StatisticsGroupId,  StatisticsItem, TravelRequestOptions} from '../index'
+import { StatisticsGroupId,  StatisticsItem} from '../index'
 import { LatLngId } from '../types'
+import { BaseRequestOptions } from '../requestOptions';
 
-export interface StatisticsTravelRequestOptions extends TravelRequestOptions {
-  // statistics: (StatisticsKey | number)[]
+export interface StatisticsTravelRequestOptions extends BaseRequestOptions {
   statisticsGroup: StatisticsGroupId
   inactiveSources?: LatLngId[]
   closestSources?: boolean
-  intersectionMode?: string
+  travelType: 'walk' | 'bike' | 'car'
```

Comment:
transit missing!

Reply:
This is on purpose, according to https://dev.route360.net/docs-public/statistics/#/Simulation_Services/post_simulation_start_ , this statistics function doesnt support transit mode.

Reply:
i talked to @DavidRedlich, the docs are incrorrect at this point. transit is supported if it is supported in the used endpoint.

Reply:
Can you please fix this in the docs @hmmmmniek ?

Reply:
I'll change this in targomo-js here on tuesday after my vacation. However, I have never done any work on the rest api docs, not sure who is responsible for that. @gerbsen